graphlearn/examples/tf/trainer.py

```diff
@@ -102,6 +192,75 @@ def _close_session():
   def run_step(self, train_ops, local_step):
     raise NotImplementedError('Use LocalTrainer or DistTrainer instead.')
 
+
+  def train_and_save(self, iterator, loss, optimizer=None, learning_rate=None,
+            epochs=10, hooks=[], save_file=None, save_ids=None, save_emb=None, **kwargs):
+    with self.context():
+      self.global_step = tf.train.get_or_create_global_step()
+      #   
+      save_iter, save_ids, save_emb = node_embedding(kwargs.get('graph'), kwargs.get('model'), kwargs.get('node_type'), kwargs.get('edge_type'))
```

Comment:
Can we just pass save_iter, save_ids, save_emb as the args of `train_and _save` and keep node_embedding in each model file?